ci/rpm7/build-dep.sh

```diff
@@ -51,16 +51,16 @@ curl -L https://github.com/mongodb/mongo-cxx-driver/archive/legacy-1.1.2.tar.gz
 && scons install --disable-warnings-as-errors --prefix=/usr/local --use-sasl-client --ssl \
```

Comment:
The upgrade done by this PR should be described in CHANGE_NEXT_RELEASE. For instance:

```
- Fix: upgrade rapidjson dependecy from 1.0.2 to 1.1.0
- Fix: upgrade microhttpd dependency from 0.9.48 to 0.9.70
```